src/main/java/com/client/core/security/tools/rsa/RSA.java

```diff
@@ -0,0 +1,102 @@
+package com.client.core.security.tools.rsa;
+
+import com.client.core.security.exception.CryptoException;
+import com.client.core.security.tools.Encryption;
+import com.google.common.base.Charsets;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.core.io.Resource;
+import org.springframework.core.io.ResourceLoader;
+import org.springframework.stereotype.Component;
+
+import javax.crypto.Cipher;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.security.KeyFactory;
+import java.security.NoSuchAlgorithmException;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.security.spec.InvalidKeySpecException;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.security.spec.X509EncodedKeySpec;
+
+/**
+ * Created by kcable on 6/1/2017.
+ *
+ * When running locally, change file location to "src/main/resources/rsa/publicKey"
+ */
+@Component
+public class RSA extends Encryption {
```

Comment:
Same here - What's the purpose of this? Also it references WEB-INF, which I believe we're getting rid of?